helios-services/src/main/java/com/spotify/helios/agent/HealthCheckerFactory.java

```diff
@@ -144,6 +146,19 @@ private static boolean compatibleDockerVersion(final DockerClient docker) {
       final int minor = Integer.parseInt(Iterables.get(split, 1, "0"));
       return major == 1 && minor >= 18;
     }
+
+    /**
+     * Gracefully handle a nonfatal issue: https://github.com/spotify/docker-client/issues/513
+     */
+    private static void handleExecStartConnectionReset(RuntimeException exception) {
+      Throwable rootCause = Throwables.getRootCause(exception.getCause());
```

Comment:
just curious, why `getRootCause(exception.getCause())` instead of `getRootCause(exception)`?